core/src/mindustry/ClientLauncher.java

```diff
@@ -182,6 +185,14 @@ public void exit(){
 
     @Override
     public void init(){
+        String dataDir = OS.env("MINDUSTRY_DATA_DIR");
+        if(dataDir != null){
+            if(Paths.get(dataDir).isAbsolute()){
```

Comment:
Do not use `Paths` or any other Java file API here. See the guidelines.

Reply:
Done, I have realised that `Files.absolute` only means `verbatim`, so that it doesn't suffer any modifications, not related to it being an absolute path. I will try to update the docstring